index.html

```diff
@@ -388,6 +388,32 @@
          as specified in [[!RFC7231]] <a href='https://tools.ietf.org/html/rfc7231#section-4.3.2'>section 4.3.2</a>.
         </p>
       </section>
+
+      <section id="httpDELETE">
+        <h2>HTTP DELETE</h2>
+        <p>
+         The <code>DELETE</code> method is optional per [[!RFC7231]] <a href='https://tools.ietf.org/html/rfc7231#section-4.3.5'>section
```

Comment:
Wouldn't it be better to state that `DELETE` is optional per the LDP specification?